extension/intents/search/search.toml

```diff
@@ -80,6 +80,17 @@ match = """
   (do a |) (search | google | look on) (google | the web | the internet |) (for |) [query] (on the web |) (for me|)
 """
 
+[[search.searchPage.followUp]]
+label = 'nextResult'
+match = """
+  (search |) next (to view |) (search |) (result{s} | item{s} | page | article |)
+""" 
+
+[[search.searchPage.followUp]]
+label = "previousResult"
+intent = 'search.previous'
+
```

Comment:
Is the idea in the second example here that it just takes the phrases from `search.previous` instead of listing them?

Reply:
Yes, that was the idea. But with allowing any matching intent be run in it's own handler, and only direct follow up phrases handled as a follow up, I think this is no longer necessary.